bridge/scripts/lib/madnetTasks.ts

```diff
@@ -217,3 +217,63 @@ task(
   const event = intrface.decodeEventLog("DepositReceived", data, topics);
   console.log(event);
 });
+
+task("scheduleMaintenance", "Calls schedule Maintenance").setAction(
```

Comment:
Can we get the Madnet factory address in this case 0x0b1f9c2b7bed6db83295c7b5158e3806d67ec5bc as a mandatory input variable?